modules/cms-kit/test/Volo.CmsKit.Application.Tests/Contents/ContentAdminAppService_Tests.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using NSubstitute;
+using Shouldly;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Volo.Abp.Domain.Entities;
+using Volo.Abp.Users;
+using Volo.CmsKit.Admin.Contents;
+using Xunit;
+
+namespace Volo.CmsKit.Contents
+{
+    public class ContentAdminAppService_Tests : CmsKitApplicationTestBase
+    {
+        private ICurrentUser _currentUser;
+        private readonly CmsKitTestData _data;
+        private readonly IContentAdminAppService _service;
+        public ContentAdminAppService_Tests()
+        {
+            _data = GetRequiredService<CmsKitTestData>();
+            _service = GetRequiredService<IContentAdminAppService>();
+        }
+
+        protected override void AfterAddApplication(IServiceCollection services)
+        {
+            _currentUser = Substitute.For<ICurrentUser>();
+            services.AddSingleton(_currentUser);
+        }
+
+        [Fact]
+        public async Task ShouldGetListAsync()
+        {
+            var result = await _service.GetListAsync(new ContentGetListInput());
+
+            result.ShouldNotBeNull();
+            result.Items.ShouldNotBeEmpty();
+            result.Items.Count.ShouldBe(4);
+        }
+
+        [Fact]
+        public async Task ShouldGetAsync()
+        {
+            var result = await _service.GetAsync(_data.Content_1_Id);
+
+            result.ShouldNotBeNull();
+        }
+
+        [Fact]
+        public async Task ShouldCreateAsync()
+        {
+            var created = await _service.CreateAsync(new ContentCreateDto
```

Comment:
We should add duplicate EntityType/EntityId creation test case.

Reply:
Can you check Commit 42eb25c3